spec/controllers/conferences_controller_spec.rb

```diff
@@ -54,4 +54,58 @@
       end
     end
   end
+
+  describe 'POST create' do
+    context 'with an unauthenticated user' do
+      let(:params) { attributes_for :conference }
+
+      it 'cannot create new conference' do
+        expect {
+          post :create,
+          params: { conference: params }
+        }.not_to change { Conference.count }
+      end
+
+      it 'cannot create new conference' do
+        expect {
+          post :create,
+          params: { conference: params },
+          xhr: true
+        }.not_to change { Conference.count }
+      end
+
+      it 'should return a 401 response status' do
+        post :create, params: { conference: params }, xhr: true
+        expect(response).to have_http_status(401)
+      end
+
+    end
+
+    context 'with user logged in' do
+      let(:student) { FactoryGirl.create(:student) }
+      let(:params) { attributes_for :conference }
+
+      before :each do
```

Comment:
Minor thing, but can you please remove the `:each` part? It's the default behavior and considered bad practise to add it 😉 
```ruby
before { sign_in student }
```
does the job

Reply:
We also have[ shared contexts](https://github.com/rails-girls-summer-of-code/rgsoc-teams/tree/master/spec/support/shared_contexts), for different kind of users. I think you can use them instead of the `before` block

Reply:
You mean it's a bad practice to use :each because I just have one context after it, or it should not be used at all?

Reply:
Sorry for causing confusion 🙈  - did not make myself clear. It's only an implicit vs. explicit thing, some linters will tell you to skip the `(:each)` part, because it's the default behavior. That's all.
It's uncommon for me to see, so I suggested to make the `before(:each)` a `before`, but it does not have any impact on the code itself.

Plus: it's of course also totally fine if you leave it like this :v:

For the shared contexts: yes, @F3PiX is right, you could use them here. But since you're setup is super minimal, I don't see a real benefit in doing so. Would probably just make things harder to read.